framework/doc/content/source/functormaterials/FunctorSmoother.md

```diff
@@ -0,0 +1,35 @@
+# FunctorSmoother
+
+!syntax description /FunctorMaterials/FunctorSmoother
+
+Three heuristics have been implemented for smoothing a functor (or several). These heuristics have
+only been implemented for the `ElemArg` functor argument. Functor arguments are explained in
+the [Functor documentation](syntax/Functors/index.md).
+
+- face averaging
+
+The elemental value of the smoothed functor is equal to the arithmetic average of the values of the previous
+functor on the faces of the element.
+
+- neighbor averaging
+
+The elemental value of the smoothed functor is equal to the arithmetic average of the values of the previous
+functor at the neighbors of the element.
```

Comment:
This one I could remove and instead rely on the smoothing layers treatment

Reply:
I ended up doing my thing just because it was the same code structure as the other. Figuring out how to make the general smoothing layers treatment would have taken me a little more time